experimental/GITFans/GITFans.jl

```diff
@@ -416,19 +438,14 @@ function fan_traversal(orbit_list, q_cone, perm_actions)
 
         # note that we run also over elements added inside the loop
         current_cone_list = cones_from_bitlist(orbit_list, current_hash)
-        intersected_cone = Polymake.polytope.intersection(current_cone_list...)
-        facets = intersected_cone.FACETS
-        facets = convert(Matrix{Rational{BigInt}},facets)
-        facet_points = Vector{Rational{BigInt}}[
-                           Polymake.polytope.facet(intersected_cone, i-1).REL_INT_POINT
-                           for i in 1:size(facets, 1)]
+        intersected_cone = intersect(current_cone_list)
+        ic_facets = -linear_inequality_matrix(facets(intersected_cone))
+        facet_points = [Vector{T}([relative_interior_point(f)...]) for f in faces(intersected_cone, dim(intersected_cone)-1)]
```

Comment:
```suggestion
        facet_points = [T[relative_interior_point(f)...] for f in faces(intersected_cone, dim(intersected_cone)-1)]
```
if #2142 is resolved then could even write
```suggestion
        facet_points = [T[relative_interior_point(f)...] for f in facets(Cone,intersected_cone)]
```